code/modules/mob/delays.dm

```diff
@@ -5,7 +5,7 @@
 // Reduces duplicated code by quite a bit.
 /datum/delay_controller
 	// Delay clamps (for adminbus, effects)
-	var/min_delay = 1
+	var/min_delay = 0.001
```

Comment:
Okay I was both wrong and right about what I said.

You can only move once per tick, thats built into byond with all of its clientside verbs like move. But try lowering the tick lag a bit and you'll see it becomes impossible to accurately move the correct number of tiles with a high modifier. The goal is to have this minimum delay a bit higher than the amount of time someone can press and depress a key normally.